README.md

```diff
@@ -67,11 +67,33 @@ after_success:
   - cover -report codecov
 ```
 
+#### GitHub Action
+
+Add to your `.github/workflows/main.yml` file
+```yml
+jobs:
+  codecov:
+    runs-on: ubuntu-latest
+    container: perl:latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run codecov GitHub Action
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+        run: |
+          cpanm --quiet --notest Module::Build Devel::Cover Devel::Cover::Report::Codecov
```

Comment:
This should be [Devel::Cover::Report::Codecovbash](https://github.com/mudler/Mojo-IOLoop-ReadWriteProcess/pull/27#issuecomment-983682803)
```suggestion
          cpanm --quiet --notest Module::Build Devel::Cover Devel::Cover::Report::Codecovbash
```